agent/lib/broker_controller.js

```diff
@@ -248,6 +281,17 @@ function is_not_internal(conn) {
     return conn.user !== undefined && conn.user.match('^agent.[a-z0-9]+$') == null; //Can't get properties or anything else on which to base decision yet
 }
 
+function compare_address_settings(orig_address_settings, new_address_settings) {
+    for (var name in orig_address_settings) {
```

Comment:
this method won't catch the case where the new settings object has a value that the old  does not.  Is this important?

Reply:
The broker returns all the properties with default values.  It is important that we only compare the ones in the new_settings.